src/markdownParser.js

```
@@ -34,10 +34,38 @@ export function parseMarkdown(content) {
+   * If the last element in the section's content is not of the specified type,
+   * a new sequence node is created and added to the content. Otherwise, the node
+   * is appended to the existing sequence of the same type.
+   *
+   * @param {Object} section - The section object containing the content array.
+   * @param {string} type - The type of the node to be added.
+   * @param {Object} node - The node to be added to the section's content.
+   */
+  const addToSequence = (section, type, node) => {
+    if (
+      section.content.length > 0 &&
+      section.content[section.content.length - 1].hasOwnProperty(type)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)